### DISCUSSION TITLE
Issue with Etsy API: Discrepancy in the number of receipts returned by API vs. shown in the interface

### DISCUSSION THREAD

**Discussion by bubrik07** [etsy/open-api#1302] [Q&A] [answered]
Hello everyone,

I’m facing an issue with the Etsy API when trying to retrieve all receipts for one of my accounts. According to the Etsy interface, the total number of receipts is around 4000, but when I use the GET /shops/{shop_id}/receipts endpoint from the Etsy API documentation, the API only returns 2300 receipts.

Here are the key details:

    I'm not using any filters in the API request.
    The request parameters and the results seem consistent, but there’s a significant difference in the total number of receipts returned compared to what I see in the Etsy interface.
    I'm using pagination to retrieve all the data, and it works as expected, but the total count doesn't match.

Questions:

    Is there a reason why the API is returning fewer receipts than what is displayed in the interface?
    Are there any hidden filters or data processing steps that I should be aware of?
    Has anyone else faced this issue with the Etsy API, and if so, how did you resolve it?

I will also attach screenshots showing the total number of receipts in the interface and the number returned via API for more clarity.

Any guidance or insights would be greatly appreciated. Thanks!

![2024-10-18 12 16 25](https://github.com/user-attachments/assets/f0c0ddc8-712d-4c37-8001-24cb17a4b7b0)
![2024-10-18 12 24 37](https://github.com/user-attachments/assets/5319426a-e0dc-42b1-8c88-5a79f74ad55e)
![2024-10-18 12 24 42](https://github.com/user-attachments/assets/3c06ee00-5297-4903-bc8a-6acd5fd5b51c)

**Comment by no1sebomb** [comment, marked as answer]
Hi! I think that these are not exactly the same values. Count of receipts is total amount of customer orders, and sales - total amount of all sold items. Some receipts can have multiple items in it, so number of sales gonna be greater than total number of receipts.

**Reply by bubrik07** [reply]
Thank you for the clarification! 
That makes sense. 
I now realize that the total number of receipts refers to the number of distinct orders, whereas the total number of sales reflects the individual items sold. 
For example, if an order contains multiple listings, it counts as one receipt but multiple sales.
